core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/wordCloud/WordCloudOpFinalExec.java

```diff
@@ -20,14 +21,21 @@
  *
  */
 public class WordCloudOpFinalExec implements OperatorExecutor {
-    private final int MAX_FONT_SIZE = 200;
-    private final int MIN_FONT_SIZE = 50;
-    private HashMap<String, Integer> termFreqMap;
+    private static final int MAX_FONT_SIZE = 200;
+    private static final int MIN_FONT_SIZE = 50;
```

Comment:
maybe better to config on the frontend?

Reply:
this is no longer needed after shifting to the new implementation